### PR TITLE
Update CMake script for the new code changes

### PR DESCRIPTION
**Pull request by tecnotercio**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -19,8 +19,15 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 if(UNIX AND NOT APPLE AND NOT DEFINED OpenGL_GL_PREFERENCE)
   set(OpenGL_GL_PREFERENCE LEGACY)
 endif()
+
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-reorder")
```
Comment (eszlari) [inline CMakeLists.txt]: Please add a comment explaining why this is needed.

File: CMakeLists.txt [hunk 2]
```diff
@@ -42,18 +49,6 @@ find_package(FFMPEG 3.4 REQUIRED
   swresample
 )
 
-find_package(frei0r)
```
Comment (eszlari) [inline CMakeLists.txt]: Remove the find module too:
https://github.com/olive-editor/olive/blob/master/cmake/Findfrei0r.cmake

File: CMakeLists.txt (hunk 1 above)
Comment (tecnotercio) [reply CMakeLists.txt]: The "-Wno-reorder" flag was inserted in the latest qmake project update (olive.pro):
https://github.com/olive-editor/olive/blob/master/olive.pro#L51

File: CMakeLists.txt (hunk 2 above)
Comment (tecnotercio) [reply CMakeLists.txt]: File removed in commit 4f54fb1

File: CMakeLists.txt (hunk 1 above)
Comment (tecnotercio) [reply CMakeLists.txt]: File changed in commit 7d0a666.

**Pull request merged**